argilla/docs/how_to_guides/dataset.md

```diff
@@ -490,4 +490,30 @@ client = rg.Argilla(api_url="<api_url>", api_key="<api_key>")
 dataset_to_delete = client.datasets(name="my_dataset")
 
 dataset_deleted = dataset_to_delete.delete()
+```
+
+## Save a dataset locally
+
+You can save an existing dataset locally by calling the `to_disk` method on the `Dataset` class.
+
+```python
+import argilla as rg
+
+client = rg.Argilla(api_url="<api_url>", api_key="<api_key>")
+
+dataset_to_save = client.datasets(name="my_dataset")
+
+dataset_saved = dataset_to_save.to_disk(path="/path/empty-directory")
```

Comment:
What does `to_disk()` returns? I'd use a more precise name for the variable and even add a sentence in the description on top about it. I also think is worth mentioning that we recommend creating an empty directory